test/src/loaders/ncbi_downloader/ncbi_downloader_test.py

```diff
@@ -1,5 +1,103 @@
+import os
+import shutil
+import uuid
+from pathlib import Path
+
+import pytest
+
 from src.loaders.ncbi_downloader import ncbi_downloader
+from src.loaders.ncbi_downloader.ncbi_downloader import GTDB_DOMAIN
+
+
+@pytest.fixture(scope="module")
+def setup_and_teardown():
+    print('starting NCBI downloader tests')
+    tmp_dir = 'result_{}'.format(str(uuid.uuid4()))
+    os.makedirs(tmp_dir, exist_ok=True)
+
+    caller_filename_full = Path(__file__).resolve()
+    caller_file_dir = os.path.dirname(caller_filename_full)
+
+    project_dir = Path(caller_file_dir).resolve().parents[3]
```

Comment:
Ok, this is the 3rd time I think I've seen this pattern, can you do a separate PR to make a test helper method in `conftest.py`